modules/tools/pom.xml

```diff
@@ -37,13 +37,23 @@
         <profile>
             <id>java-9+</id>
```

Comment:
Do we still want to call this profile `java-9+` (as it is not going to include 15+ anymore)? Wouldn't something like `java-9-14` more honest? Or do we want to be 'backward-compatible' with profile naming?